index.html

```diff
@@ -1,276 +1,249 @@
-                            <li>Html</li>
-                            <li>CSS</li>
-                            <li>Javascript</li>
-                        </ul>
-                        <a class="project_click_class text_body_font">
-                            <p> See this project</p>
-                            <div class="arrow_key"></div>
-                        </a>
-                    </div>
-
-                    <div class="grid_3 grid_picture mobile_off"></div>
-                    <div class="grid_row mobile_off">
-                        <p class="pro_text text_body_font">Tonic</p>
-                        <ul class="list_category text_body_font">
-                            <li >Html</li>
-                            <li >CSS</li>
-                            <li >Javascript</li>
-                        </ul>
-                        <a class="project_click_class text_body_font">
-                            <p> See this project</p>
-                            <div class="arrow_key"></div>
-                        </a>
-                    </div>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Emmanuel Portfolio</title>
+
+  <link rel="stylesheet" href="./css/index.css" />
+  <link rel="stylesheet" href="./css/works.css" />
+  <link rel="stylesheet" href="./css/about.css" />
+  <link rel="stylesheet" href="./css/contact.css" />
+  <link rel="stylesheet" href="/css/desktop.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+
+<body>
+  <div class="flex_container">
+    <header>
+      <div class="mobile_logo mobile_off"></div>
+      <nav class="flex_nav text_body_font text_color">
+        <img src="./img/shape-1.svg" class="shape-1" alt="">
+        <img src="./img/shape-2.svg" class="shape-2" alt="">
+        <div class="mobile-icons">
+          <i class="fa fa-bars fa-xl hamb"></i>
+          <i class="fa fa-xmark fa-solid fa-xl close-icon"></i>
+        </div>
+        <img src="./img/img-before.svg" class="img-before" alt="">
+        <ul class="mobile-options">
+          <li class="li_first"><a href="#hello">Hello</a></li>
+          <li class=""><a href="#portfolio">Portfolio</a></li>
+          <li class=""><a href="#about">About</a></li>
+          <li class=""><a href="#contact">Contact</a></li>
+        </ul>
+
+        <ul>
+          <li class="li_first"><a href="#hello" id="hello">Hello</a></li>
+          <li class=""><a href="#portfolio">Portfolio</a></li>
+          <li class=""><a href="#about">About</a></li>
+          <li class=""><a href="#contact">Contact</a></li>
+        </ul>

```

Comment:
- [ ] Please make sure that the mobile hamburger menu button only appears on the mobile view and not the desktop view. Currently, the hamburger menu is displayed on both the mobile and desktop views.
![image](https://user-images.githubusercontent.com/62290545/163203451-310969a4-0179-46e3-a587-804a06d7b686.png)